pkg/bootstrap/config.go

```diff
@@ -118,16 +119,35 @@ func (cfg Config) toTemplateParams() (map[string]any, error) {
 	opts = append(opts, getNodeMetadataOptions(cfg.Node)...)
 
 	// Check if nodeIP carries IPv4 or IPv6 and set up proxy accordingly
```

Comment:
I am wondering that dual stack is not well protected under the feature flag  `EnableDualStack`.  I think we should improve

Reply:
@howardjohn @ramaraochavali Ideally it should not carry ipv6 address at all if the feature disabled.  Same rule should also apply to iptables interception, i haven't see if it was there

Reply:
> I am wondering that dual stack is not well protected under the feature flag `EnableDualStack`. I think we should improve

Hi @hzxuzhonghu, yeah, it makes sense, should protect this with feature flag `EnableDualStack`. Thanks for the remind!

Reply:
> I am wondering that dual stack is not well protected under the feature flag `EnableDualStack`. I think we should improve

Hi @hzxuzhonghu, yeah, it makes sense, should protect this with feature flag, however, `EnableDualStack` is the control plane's flag. And thanks for the remind!

Reply:
@hzxuzhonghu,  as you said, I added the feature flag to protect the original logic if dual stack is disable.